src/job.rs

```diff
@@ -737,10 +737,10 @@ async fn add_all_recipients_as_contacts(context: &Context, imap: &mut Imap, fold
         Ok(contacts) => {
             let mut any_modified = false;
             for contact in contacts {
-                let display_name_normalized = contact
+                let display_name_normalized: String = contact
```

Comment:
Why this change?

Reply:
Removed it, it was needed at some point when compiler wasn't able to figure the type out until I turned more `AsRef<str>` into `&str`.